src/index.js

```diff
@@ -0,0 +1,8 @@
+const stipulate = {
+  working() {
+    return true;
+  }
+};
+
+export default stipulate;
+module.exports = stipulate;
```

Comment:
we could, but that means es5 consumers of this lib have to:
```js
var stipulate = require('stipulate').default;
```
instead of:
```js
var stipulate = require('stipulate');
```

another option is a [plugin](https://github.com/59naga/babel-plugin-add-module-exports).